views/js/review/plugins/content/item-answer/item-answer.js

```diff
@@ -256,144 +218,67 @@ define([
 
             /**
              * Defines the item status
-             * @param {String} status
+             * @param {String} statusType
+             * @param {Boolean} hasCorrectResponseTab
              * @returns {itemAnswerComponent}
              * @fires statuschange
              */
-            setStatus(status) {
-                const change = this.getConfig().status !== status;
+            setStatus(statusType, hasCorrectResponseTab) {
+                let status;
+                if (statusType === 'info') {
```

Comment:
Much better than all of the setters/getters. Thanks!

Any chance to unify `statusType` and `status` values? Just asking.

Reply:
Ok, let's do this.
Originally I wanted to use same values as in [review-panel sidebar](https://github.com/oat-sa/extension-lti-test-review/blob/448c078429b112a9b3da60efa3323338c86ac906/views/js/review/plugins/navigation/review-panel/panel-data.js#L58-L85) and extract a reusable method to some shared file, but at the moment yes, it's cleaner to just rename them.

Reply:
Done, thanks for suggestion.